pkg/storage/utils/eosfs/eosfs.go

```diff
@@ -119,6 +118,14 @@ func (c *Config) init() {
 		c.UserLayout = "{{.Username}}" // TODO set better layout
 	}
 
+	if c.UserIDCacheSize == 0 {
+		c.UserIDCacheSize = 1000000
+	}
+
+	if c.UserIDCacheWarmupDepth == 0 {
+		c.UserIDCacheWarmupDepth = 2
```

Comment:
Perhaps put a comment about what depth means in terms on layout for example for CERN: `/eos/user/<letter>/<username>`...

Reply:
Will add